### DISCUSSION TITLE
Fast Riemann Zeta computation

### DISCUSSION THREAD

**Discussion by Bobingstern** [josdejong/mathjs#2850] [Ideas]
Math.js should provide a fast implementation for the zeta function. I have tried doing this in JavaScript before and somewhat succeeded in the [repo](https://github.com/Bobingstern/special.js#riemann_zetas-complex-prec-number) although it would great if someone could figure out how to use Odlyzko and Schonage algorithm for multi evaluations of the zeta function making use of the FFT of all things! The only thing describing this is the [paper](https://www.ams.org/journals/tran/1988-309-02/S0002-9947-1988-0961614-2/S0002-9947-1988-0961614-2.pdf) although the math is fairly heavy. This was the same algorithm used to compute the first trillion zeros of the function. I tried and failed trying to implement this so maybe someone with more numerical analysis experience and knowledge could figure it out.

**Comment by josdejong** [comment]
That sounds like an interesting challenge :) Thanks for the pointer to the paper.

Anyone interested to give it a try?
